ofctrl/transaction.go

```diff
@@ -22,14 +22,15 @@ func (t TransactionType) getValue() uint16 {
 }
 
 const (
-	Atomic  = TransactionType(openflow13.OFPBCT_ATOMIC)
-	Ordered = TransactionType(openflow13.OFPBCT_ORDERED)
+	Atomic  = TransactionType(openflow15.OFPBCT_ATOMIC)
+	Ordered = TransactionType(openflow15.OFPBCT_ORDERED)
 )
 
 var uid uint32 = 1
 
 type OpenFlowModMessage interface {
 	resetXid(xid uint32) util.Message
+	getXid() uint32
```

Comment:
Since you add a new signature (`getXid`) in interface `OpenFlowModMessage`, please remember to implement it in all of FlowBundleMessage, GroupBundleMessage, and MeterBundleMessage. I didn't find the implementation in MeterBundleMessage in fact.

Reply:
Added in MeterBundleMessage.